front/assets/style/base.scss

```diff
@@ -102,11 +107,20 @@
 
 /* dark theme styles */
 .theme--dark.v-btn {
-  background: $primary-color !important;
-  color: white !important;
+  background-color: $btn-default-background !important;
+  padding: 11px 24px !important;
+  color: $btn-default-color !important;
+  font-family: Avenir LT Std;
 
   &:hover {
-    background: $primary-color !important;
+    height: 46px !important;
+  }
+  &:focus {
+    height: 46px !important;
+    background-color: $btn-focus-background !important;
```

Comment:
'buttons': (
      'disabled': rgba(map-get($shades, 'black'), 0.26),
      'focused': rgba(map-get($shades, 'black'), 0.12),
      'focused-alt': rgba(map-get($shades, 'white'), 0.6),
      'pressed': rgba(#999, 0.4)
    ),

Reply:
@aldarund these don't seem to work.
only  `focused` reflects on the button, but it only works when the button is `disabled` so that is incorrect.
the rest have no effect.

Please test here
https://codesandbox.io/s/determined-rosalind-pfhx0?file=/pages/index.vue